### PR TITLE
Update default Rust version to 1.48.0.

### PR DESCRIPTION
**Pull request by PiotrSikora**
Signed-off-by: Piotr Sikora <piotrsikora@google.com>

### CODE REVIEW

File: rust/known_shas.bzl [hunk 1]
```diff
@@ -155,23 +155,27 @@ FILE_KEY_TO_SHA = {
...
     "2020-11-10/rust-std-nightly-wasm32-wasi": "baf705571736331dd5449e1473b590477e2a48ad0adc6a897516fb8f1a5780fd",
     "2020-11-10/rust-std-nightly-x86_64-apple-darwin": "2b0d1758c20fea48e8afa5c9cc2844e9eb5c77376992f2af1e68261e1b0bd773",
+    "2020-11-10/rust-std-nightly-x86_64-pc-windows-msvc": "9e5f1089de87e9d54417038f7d5d30de5e604bf82f5be557361cd02b55abb018",
     "2020-11-10/rust-std-nightly-x86_64-unknown-freebsd": "24a506f85e178be0799d12f9354c6129004068552c8f5321a519b033631b815d",
     "2020-11-10/rust-std-nightly-x86_64-unknown-linux-gnu": "367c14b7fbe98e264b0e4b5a9ddaf3f78ce3ce09bbef4c7be33a3f2abade9ad9",
     "2020-11-10/rustc-nightly-aarch64-apple-darwin": "a461f2486013b5cec450c8f79230e83878689b803a38df7304adea27b025ef1b",
     "2020-11-10/rustc-nightly-aarch64-unknown-linux-gnu": "900170006c4c2d88cadc0d915d410588cb80150817e53aa7fca41a459a5ec500",
     "2020-11-10/rustc-nightly-x86_64-apple-darwin": "7a443dfb068bb7e3854dd6475564da33a57d3f225ce03ad8bc973e8900960b69",
+    "2020-11-10/rustc-nightly-x86_64-pc-windows-msvc": "e7b325e55d372aaf4be400273673711fe78271b655c0b710d62a972b8044b9ef",
```
Comment (PiotrSikora) [inline rust/known_shas.bzl:178]: Note: those are unrelated to this PR, but it looks like there were missing previously due to a race between PRs.

**Review by mfarrugi — APPROVED** [review]

**Comment by PiotrSikora** [comment]
(Rebased on top of current `master` branch)

**Comment by jmillikin** [comment]
Gentle ping -- looks like it might have gotten stuck behind a flaky test?

**Comment by PiotrSikora** [comment]
Fixed, let me know if you want me to squash the commits or if you can do it yourself during merge.

@jmillikin Thanks for the ping, I've missed the `clippy` errors.

**Comment by UebelAndre** [comment]
@damienmg would you be able to take a look here? 🙏

**Comment by illicitonion** [comment]
We finally got there! 🎉

**Pull request merged**